glide-core/src/protobuf/redis_request.proto

```diff
@@ -266,13 +267,21 @@ message Transaction {
     repeated Command commands = 1;
 }
 
+message ClusterScan {
+    string cursor = 1;
+    optional string match_pattern = 2;
+    optional int64 count = 3;
+    optional string object_type = 4;
```

Comment:
sorry I deleted the comment before I saw you responded. I think adding an enum in protobuf might not be the best solution as we already have an enum in the core side and an enum in the wrapper side, it would just add an overhead to add another one in protobuf.
Ideally we would have only a single enum in the core side - and probably another one in protobuf, however because this enum is a user-facing API, we don't want to expose the protobuf enum as a part of the API, and importing the redis-rs obejctType would also require to duplicate each enum in the lib.rs file to map from the ffi-enum to the rust-enum, which wouldn't save us anything. So we probably best keeping it as you did with all other user exposed command options.